### ISSUE TITLE
Options object is mutated

### ISSUE THREAD

**Issue by darrenscerri** [leizongmin/js-xss#66]
```var opts = {};
var oldKeys = Object.keys(opts);
xss('test', opts);
var newKeys = Object.keys(opts);

oldKeys.length === newKeys.length
```

Re-using an opts which has `stripIgnoreTag` defined, will issue a warning:

```Notes: cannot use these two options "stripIgnoreTag" and "onIgnoreTag" at the same time```

**Comment by leizongmin** [comment]
Hi, I just fixed this problem in the new version `xss@0.3.3`, please try the latest version.

**Comment by darrenscerri** [comment]
Looks good, thanks!
